docs/onboard-public-officers-using-non-se-machines.md

```diff
@@ -1,17 +1,16 @@
-# Onboarding public officer using non-SE GSIB device
+# Onboard public officers
 
-This article guides public officers to sign up for TechPass account and to onboard to it using their non-SE GSIB machine.
-
-While public officers create their TechPass using non-SE GSIB machine, they can use their TechPass account to access Singapore Tech Stack(SGTS) products and services from GSIB and GMD machines.
+This article guides public officers to sign up for TechPass account and to onboard to it using their non-SE GSIB machine. They can use their TechPass account to access Singapore Tech Stack(SGTS) products and services from GSIB and GMD machines.
```

Comment:
The full name of SGTS is Singapore Government Tech Stack
https://www.tech.gov.sg/products-and-services/singapore-government-tech-stack/

Reply:
Done, thank you!

Reply:
On the home.md page it is mentioned as Technology Stack, so I too will use the same for consistency.

Reply:
Singapore Government Technology Stack (SGTS)